src/sisl/io/orca/txt.py

```diff
@@ -147,5 +147,119 @@ def read_geometry(self):
 
         return Geometry(xyz, atoms)
 
+    @sile_fh_open()
+    def read_gtensor(self):
+        """Reads electronic g-tensor data from the EPRNMR_GTensor block
+        in the ORCA property txt file
+
+        Returns
+        -------
+        PropertyDict : Electronic g-tensor
+        """
+        G = PropertyDict()
+        f, line = self.step_to("EPRNMR_GTensor", allow_reread=False)
+        if not f:
+            return None
+        for i in range(5):  # step 5 lines ahead
+            v = self.readline()
+        G["multiplicity"] = int(v.split()[-1])
+        self.readline()
+        self.readline()
+        tensor = np.empty([3, 3], np.float64)
+        for i in range(3):
+            v = self.readline().split()
+            tensor[i] = v[1:4]
+        G["tensor"] = tensor  # raw (total) tensor
+        self.readline()  # skip g tensor line
+        self.readline()  # skip header
+        vectors = np.empty([3, 3], np.float64)
+        for i in range(3):
+            v = self.readline().split()
+            vectors[i] = v[1:4]
+        G["vectors"] = vectors  # g-tensor eigenvectors
+        self.readline()  # skip eigenvalue line
+        self.readline()  # skip header
+        v = self.readline().split()
+        G["eigenvalues"] = np.array(v[1:4], np.float64)
+
+        return G
+
+    @sile_fh_open()
+    def read_hyperfine_coupling(self):
+        """Reads hyperfine couplings from the EPRNMR_ATensor block
```

Comment:
I will also adapt these small things:

1. when having math, one needs to use the `r"""` prefix to tell it to sustain the `\` in the code. `r` for `raw`.
2. I will change `\text` to `\mathrm` and `\boldsymbol` to `\mathbf`

Reply:
OK, you may then also take a look at the docstring in `sisl.physics.electron.berry_curvature` that I followed.

Reply:
Thanks! 
The difference between `boldsymbol` and `mathbf` is that `boldsymbol` can be used to bold a greek letter in a consistent manner. AFAIK boldsymbol on letters is just making them bold, but not upright. `mathbf` is bold *and* uprighting the letters, but does nothing for greek letters. So `\boldsymbol` shold be used for greek letters, otherwise, `\mathbf`. I just checked `berry_curvature`, it seems consistent with what I wrote (unless I missed something).

Anyway, thanks!